cran-comments.md

```diff
@@ -1,5 +1,36 @@
 # CRAN Submission History
 
+## v 0.4.0
+
+### Submission on April 1st, 2019
+This is a minor release that adds vignette creation functionality and makes a number of other minor changes and bug fixes.  Please see `NEWS.md` for details.
+
+As always, issues are tracked and source code is maintained at https://github.com/UptakeOpenSource/pkgnet.
+
+### CRAN Response on April 2nd, 2019
+Thanks, we see:
+
+   Found the following (possibly) invalid URLs:
+   ...
+   [two in the CreatePackageVignette()]
+   [one in NEWS.md]
+   ...
+
+Please fix and resubmit.
+
+
+Is there some reference about the method you can add in the Description 
+field in the form Authors (year) <doi:.....>?
+
+### Resubmission on April 2nd, 2019 (planned)
+The problematic links have been fixed:   
+* two were non-canonical CRAN links and   
+* one was a link to an upcoming gallery exhibit that does not yet exist.   
+
+I had seen the warnings for the non-canonical CRAN links during testing and had interpreted them as optional.  In future testing, these CRAN link formatting warnings will be interpreted as required.  However, the broken link had gone unnoticed, burred under the other two warnings.  Thanks for the check.  
+
+Regarding the references comment, the package is original work aside from the functionality utilized from the packages listed in the DESCRIPTION file.  However, I will confer with the pkgnet authors on writing a paper of our own, describing our motivations and design decisions in detail (with a full bibliography).  Thanks for the complement! 
```

Comment:
Compliment with an e. Also I don't think this was a compliment, that's just a standard thing they started asking for recently.